pxc/local/build-binary-pxc57

```diff
@@ -69,6 +68,8 @@ if [[ "${WITH_ASAN}" = "yes" ]]; then
     CMAKE_OPTS+=" -DWITH_ASAN=ON"
     BUILD_COMMENT+="-asan"
 fi
+export CMAKE_OPTS=${CMAKE_OPTS}
+export SCONS_ARGS=${SCONS_ARGS}
```

Comment:
What about exporting other variables like `BUILD_COMMENT` that are used by `build-binary.sh`?

Reply:
@kamil-holubicki The other remaining variables along with BUILD_COMMENT don't come from the Jenkins job parameters and they are computed again in build-binary.sh. But, I see that "-asan" is not added to the comment properly in build-binary.sh. So, I've exported BUILD_COMMENT and the rest as well before we remove the duplications(hopefully in the future).